pkg/lib/interaction/intents/authenticate.go

```diff
@@ -357,6 +357,10 @@ func (i *IntentAuthenticate) DeriveEdgesForNode(graph *interaction.Graph, node i
 			panic(fmt.Errorf("interaction: unexpected authentication stage: %v", node.Stage))
 		}
 	case *nodes.NodePromptCreatePasskeyEnd:
+		return []interaction.Edge{
+			&nodes.EdgeConfirmTerminateOtherSessionsBegin{},
+		}, nil
```

Comment:
Currently, when I try to promote an anonymous user in the mobile app, I will see this warning screen. Since a refresh token was issued when the user is an anonymous user. Maybe we should only show this when login?